File: fleet-service/src/main/java/com/fleet/Order.java
package com.fleet;

public class Order {
    private FoodOrder foodOrder;
    private Drone drone;

    public Order(FoodOrder foodOrder, Drone drone) {
        this.foodOrder = foodOrder;
        this.drone = drone;
    }

    public FoodOrder getFoodOrder() {
        return foodOrder;
    }

    public void setFoodOrder(FoodOrder foodOrder) {
        this.foodOrder = foodOrder;
    }

    public Drone getDrone() {
        return drone;
    }

    public void setDrone(Drone drone) {
        this.drone = drone;
    }
}

File: operating-web-client/src/components/Order.js
import React, { useState, useEffect } from 'react';
import { fetchOrders } from '../services/fleet';
import '../styles/Order.css'


export const OrderList = () => {
    const [ orderList, setOrderList ] = useState([]);

    useEffect(() => {
        async function fetchData() {
            const data = await fetchOrders();
            setOrderList(data);
        }
        fetchData();
    }, [])

    return (
        <div className="order-list">
          {orderList.map((order) => (
            <Order
              key={order.id}
              status={order.status}
              drone={order.drone}
              restaurantLocation={order.restaurantLocation}
              customerLocation={order.customerLocation}
              foodOrder={order.foodOrder}
              createdAt={order.createdAt}
            />
          ))}
        </div>
    );
}


export const Order = ({ status, drone, restaurantLocation, customerLocation, foodOrder, createdAt }) => {

    return (
        <div className="order">
          <p>Status: {status}</p>
          <p>Drone: {drone}</p>
          <p>Restaurant Location: {restaurantLocation}</p>
          <p>Customer Location: {customerLocation}</p>
          <p>Food Order: {foodOrder}</p>
          <p>Created At: {(new Date(createdAt)).toUTCString()}</p>
        </div>
    );

}